src/queries.js

```diff
@@ -64,6 +64,14 @@ export const resourceInfoFragment = gql`
       id
       name
     }
+    meta {
```

Comment:
Only use meta in subjectpage. This will make all queries using resourceInfo slower.